### ISSUE TITLE
Light version?

### ISSUE THREAD

**Issue by caobo** [azemoh/vscode-one-monokai#13]
This is really nice, do you also plan to make a light version?

**Comment by FunkyloverOne** [comment]
Yeah, and many thanks for this *(finally)* really good theme for Ruby! :heart: :hugs:
